_docs/_latest/configure/notifier/index.md

```diff
@@ -234,24 +234,18 @@ the UI or API:
 - Note: You will need to have either owner, editor or service management roles
   in the Project in order to enable the API
 
-1. Enable the  API connector config to Cloud SCC.  Specifically, this means
-in the Forseti project server bucket, edit the `configs/forseti_conf_server.yaml`,
- as follows:
+1. Enable the  API connector config to Cloud SCC by configuring the following fields in 
```

Comment:
What about the path for using deployment manager?

Reply:
Good point- please take look at updated.